db/handle_buf.c

```diff
@@ -915,7 +915,7 @@ int handle_buf_main2(struct dbenv *dbenv, SBUF2 *sb, const uint8_t *p_buf,
     int numwriterthreads;
     struct dbq_entry_t *newent = NULL;
 
-    if (db_is_stopped()) {
+    if (db_requests_are_stopped()) {
```

Comment:
if (db_is_stopped() || db_requests_are_stopped()) ?

Reply:
Sure.  req_thds should all be gone at this point by virtue of calling stop_threads, but I can add that regardless.